src/SFA.DAS.FAT.Domain/Courses/QarModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+namespace SFA.DAS.FAT.Domain.Courses;
+
+public sealed class QarModel
+{
+    public string Period { get; set; }
+    public string Leavers { get; set; }
+    public string AchievementRate { get; set; }
+    public string NationalLeavers { get; set; }
+    public string NationalAchievementRate { get; set; }
+    public int TotalParticipants => TotalParticipantCount();
+    public decimal FailureRate => GetFailureRate();
+    public string QarPeriodStartYear { get => $"20{Period.AsSpan(0, 2)}"; }
+    public string QarPeriodEndYear { get => $"20{Period.AsSpan(2, 2)}"; }
+    public string PeriodDisplay => $"{QarPeriodStartYear} to {QarPeriodEndYear}";
+
+    public int ConvertedLeavers
```

Comment:
this is cleaner, and you can use the pattern elsewhere in this file
return int.TryParse(Leavers, out var value) ? value : 0;